eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -195,15 +195,24 @@ stages:
                           OpenAsDraft: ${{ parameters.TestPipeline }}
                           CloseAfterOpenForTesting: '${{ parameters.TestPipeline }}'
 
+                      - task: AzureCLI@2
+                        displayName: Authenticate to OpenSource API
+                        condition: and(succeeded(), eq(variables['PublishToVcpkg'], 'true'))
+                        inputs:
+                          azureSubscription: opensource-api-connection
+                          scriptType: pscore
+                          scriptLocation: inlineScript
+                          inlineScript: |
+                            $jwt_accessToken = az account get-access-token --resource "api://2789159d-8d8b-4d13-b90b-ca29c1707afd" --query "accessToken" --output tsv
+                            Write-Host "##vso[task.setvariable variable=opensource-api-token;isSecret=true]$jwt_accessToken"
```

Comment:
We write the access token to the build log? Is it safe?

Reply:
That doesn't show up in the logs it is a special DevOps logging command and it is marked as secret so if it got dumped anywhere else it would get starred out in the logs.

Reply:
Thank you! I can confirm, I looked at the logs, and many values are displayed as `***`.